rtgui/filmsimulation.cc

```diff
@@ -64,6 +64,10 @@ FilmSimulation::FilmSimulation()
     :   FoldableToolPanel( this, "filmsimulation", M("TP_FILMSIMULATION_LABEL"), false, true )
 {
     m_clutComboBox = Gtk::manage( new ClutComboBox(options.clutsDir) );
+	// The following line, inspired by https://stackoverflow.com/questions/43414163/gtk-mm-limit-width-of-combobox,
+	// makes the Film Simulation ComboBox look better if there are HaldCLUT names that are too long to fit into it
+	((Gtk::CellRendererText*)m_clutComboBox->get_first_cell())->property_ellipsize() = Pango::ELLIPSIZE_END;
```

Comment:
Rather than cast the result of `get_first_cell()` into a `CellRendererText*`, it would be better, if possible, to explicitly create a `CellRendererText`, set its ellipsize property, and add it to the combo box.

Currently, we are packing the model column.
https://github.com/Beep6581/RawTherapee/blob/bab5516871ebe0eca63e9f97689468f6de0a818c/rtgui/filmsimulation.cc#L221
You can replace this with a new `CellRendererText` and set its text property, with [add_attribute](https://developer-old.gnome.org/gtkmm/3.16/classGtk_1_1CellLayout.html#ae5f84edaa30e5c586ca9266961bb4f96) for example.